org.lflang/META-INF/MANIFEST.MF

```diff
@@ -18,7 +18,8 @@ Require-Bundle: org.eclipse.xtext,
  com.google.guava,
  org.eclipse.xtend.lib.macro,
  org.apache.commons.cli;bundle-version="1.4",
- org.jetbrains.kotlin.bundled-compiler;resolution:=optional
+ org.jetbrains.kotlin.bundled-compiler;resolution:=optional,
```

Comment:
That line was already there. The line changed because a comma was added. I'm not sure why we have that dependency, but since it's listed as optional it doesn't look too alarming. @cmnrd could you comment on this?

Reply:
If you remove this dependency, Kotlin classes will stop working in Epoch.

Other than normal Java applications, Eclipse does not have a global class loader, instead it uses per plugin class loaders that are populated based on the manifest files (OSGI). Hence, these dependencies decide which classes are available at compile/runtime. Since the complied Kotlin classes depend on a Kotlin framework library at runtime and the `org.jetbrains.kotlin.bundled-compiler` seems to be the only plugin that provides this library, it is crucial.
Its resolution is marked optional because we do not have Kotlin support in the developer IDE and hence this dependency can only be resolved in Epoch and should not raise an error in the IDE.

Reply:
Thanks for clarifying! Is there a way to comment the dependencies in the manifest? I think this would be really helpful to prevent future discussions and to see why certain dependencies are there.

Reply:
As far as I know, [manifest files](https://docs.oracle.com/javase/6/docs/technotes/guides/jar/jar.html#JAR%20Manifest) do not support comments.